drv/transceivers-server/src/udp.rs

```diff
@@ -365,18 +386,33 @@ impl ServerImpl {
                     );
                 }
 
-                let result = self.read(read, mask & !self.disabled, out);
-                ringbuf_entry!(Trace::OperationResult(result));
+                let (data_len, result) =
+                    if self.front_io_board_present.unwrap_or(false) {
+                        let r = self.read(read, mask & !self.disabled, out);
+                        let len = r.success().count() * read.len() as usize;
+                        (len, r)
+                    } else {
+                        (
+                            0,
+                            ModuleResult::new(
+                                LogicalPortMask(0),
+                                LogicalPortMask(0),
+                                mask,
+                                LogicalPortFailureTypes::default(),
+                            )
+                            .unwrap(),
```

Comment:
and this probably also should be `unwrap_lite`
```suggestion
                            .unwrap_lite(),
```